lib/containers/container_images.pm

```diff
@@ -175,6 +175,32 @@ sub test_opensuse_based_image {
     }
 }
 
+sub verify_userid_on_container {
+    my ($runtime, $image) = @_;
+    record_info "host uid", script_output "echo \$UID";
+
+    record_info "root user inside", "process runs in the user's host uid";
+    my $cid = script_output "$runtime run -d --rm --name test1 $image sleep infinity";
+    assert_script_run "$runtime exec -it $cid id";
+    assert_script_run "$runtime exec -it $cid cat /proc/self/uid_map";
+    validate_script_output "podman top -l user huser", sub { /root\s+1000/ };
```

Comment:
Is it intentional to mix up `$runtime` and `podman` here?

Reply:
no it is not. fixed